demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -496,6 +497,13 @@ def find_core_packs_depend_on_non_core_packs(
             self._add_relationships_to_objects(session, results)
             return [self._id_to_obj[result] for result in results]
 
+    def find_mandatory_hidden_packs_dependencies(self, pack_ids):
+        with self.driver.session() as session:
+            results = session.execute_read(validate_hidden_pack_dependencies, pack_ids)
+            self._add_nodes_to_mapping(result.node_from for result in results.values())
+            self._add_relationships_to_objects(session, results)
+            return [self._id_to_obj[result] for result in results]
```

Comment:
can unindent these 3 lines, since `results` is assigned to above. Not too important, tho